src/snipe/cli/cli_qc_parallel.py

```
@@ -0,0 +1,617 @@
+import sys
+import time
+import logging
+from typing import Optional, Any, List, Dict, Set, Union
+
+import click
+import pandas as pd
+from tqdm import tqdm
+import concurrent.futures
+
+from snipe.api.enums import SigType
+from snipe.api.sketch import SnipeSketch
```

Ruff

`snipe.api.sketch.SnipeSketch` imported but unused

Remove unused import: `snipe.api.sketch.SnipeSketch`

(F401)